plugins/builddep.py

```diff
@@ -94,9 +94,9 @@ def configure(self):
         # enable source repos only if needed
         if not (self.opts.spec or self.opts.srpm):
             for pkgspec in self.opts.packages:
-                if not (pkgspec.endswith('.src.rpm')
-                        or pkgspec.endswith('nosrc.rpm')
-                        or pkgspec.endswith('.spec')):
+                if not (pkgspec.endswith('.src.rpm') or
```

Comment:
I reverted change of breaking around binary operators. And added it to flake ignore